app/Scrambler.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable-all
+
+from random import choice, shuffle
+
+from Picture import Picture
+from Tile import Tile
+
+
+class Scrambler:
+    @classmethod
+    def scramble_rows(cls, picture: Picture) -> None:
+        """Splits an Image up into rows an rearranges them."""
+        num_tiles = 8
+        image_size = picture.image.size
+        tile_size = (image_size[0], int(image_size[1] / num_tiles))
+
+        for y in range(num_tiles):
+            x0, y0, x1, y1 = (0, tile_size[1] * y, tile_size[0], tile_size[1] * (y + 1))
+            picture.tiles[y] = Tile(picture.image.crop((x0, y0, x1, y1)), (x0, y0))
+
+        picture.tile_order = list(picture.tiles.keys())
+        shuffle(picture.tile_order)
+
+    @classmethod
```

Comment:
The scrambler functions do not rely on any properties of the class. The class is just there to group them since they have a similar purpose. This is typically the job of a module. 

The scramblers could be refactored to just be functions as part of a module, which would simplify their use. What do you think?

Reply:
I agree with this, since they're all static/classmethods, functions work just fine.

Reply:
Made the class changes now, #9 is now resolved due to this